docs/topics/db/sql.txt

```diff
@@ -4,8 +4,33 @@ Performing raw SQL queries
 
 .. currentmodule:: django.db.models
 
-When the :doc:`model query APIs </topics/db/queries>` don't go far enough, you
-can fall back to writing raw SQL. Django gives you two ways of performing raw
+.. Important:: 
+    
+    As of Django 2.1, there should be no reason to have to drop into raw SQL to 
```

Comment:
I'd avoid hardcoding 2.1 here. There's also still valid reasons to use `raw()` so maybe _little reasons_ would be more appropriate?